MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -1,13 +1,73 @@
 using System;
 using System.Collections.Generic;
-using System.Globalization;
+using System.Collections.ObjectModel;
 using System.Linq;
 using System.Text.RegularExpressions;
 using MediaBrowser.Model.Diagnostics;
 using Microsoft.Extensions.Logging;
 
 namespace MediaBrowser.MediaEncoding.Encoder
 {
+    public class FFmpegVersion
+    {
+        private readonly int _major;
```

Comment:
Why not use the `Version` class?

Reply:
@ploughpuff Any thoughts on this one? @Bond-009 is this your only blocker on this one?

Reply:
Modified to using Version class.